config/presets/demo_preset.json

```diff
@@ -0,0 +1,88 @@
+{
+  "title": "Preset for Demonstration Suite",
+  "id": "demo_preset",
```

Comment:
One thing our current preset system does is allow you to assign a relevant host for the preset, which gets filtered at runtime by the server to only show presets relevant to that host.  We do this so that our `inferno-qa` site can have just a preset for the reference server on `inferno-qa`.   In theory, it is also useful for the cases when you need different client ids based on domains etc.  Is this feature addressed?  Ok to push off for later consideration...

Reply:
I think we can push that off. I think it's easier to manage individual presets now, so we may not end up needing this feature.